content/docs/command-reference/gc.md

```diff
@@ -20,14 +20,20 @@ DVC cache but are no longer needed. With `--cloud` it also removes data in
 To avoid accidentally deleting data, it raises an error and doesn't touch any
 files if no scope options are provided. It means it's user's responsibility to
 explicitly provide the right set of options to specify what data is still needed
-(so that DVC can figure out what fils can be safely deleted).
+(so that DVC can figure out what files can be safely deleted).
 
 One of the scope options (`--workspace`, `--all-branches`, `--all-tags`,
 `--all-commits`) or a combination of them must be provided. Each of them
 corresponds to keeping the data for the current workspace, and for a certain set
 of commits (determined by reading the DVC-files in them). See the
 [Options](#options) section for more details.
 
+> Please note that `dvc gc` pulls
+> [`.dir` cache](/doc/user-guide/dvc-files-and-directories#structure-of-cache-directory)
+> files for corresponding directories from remote. It pulls them when they are
+> missing and referenced in the commits that `gc` analyzed in `.dvc` or
+> `dvc.yaml` files.
```

Comment:
This note is directly about `dvc gc` so no need for the block quote (no `>`s please).

Reply:
Actually no... It's kind of a very technical implementation detail so the block quotes can stay. Never mind!